Intermittent test failures expecting 201

Some of the tests assert that a `201` status code is received to validate that a document was created.
These tests have been observed to fail under circumstances when a `202` response code is received. Any `2XX` code should be sufficient for these tests: 

>com.cloudant.tests.ResponseTest > verifyDocumentSaved FAILED
    java.lang.AssertionError: expected:<201> but was:<202>

>com.cloudant.tests.ResponseTest > verifyBulkDocumentRequest FAILED
    java.lang.AssertionError: expected:<201> but was:<202>

>com.cloudant.tests.ResponseTest > verifyDocumentConflict FAILED
    java.lang.AssertionError: expected:<201> but was:<202>

>com.cloudant.tests.UnicodeTest > testEscapedUnicode FAILED
    java.lang.AssertionError: expected:<201> but was:<202>

>com.cloudant.tests.UnicodeTest > testLiteralUnicode FAILED
    java.lang.AssertionError: expected:<201> but was:<202>

--- a/src/test/java/com/cloudant/tests/ResponseTest.java
+++ b/src/test/java/com/cloudant/tests/ResponseTest.java
@@ -52,7 +52,7 @@ public class ResponseTest {
     @Test
     public void verifyDocumentSaved() {
         Response response = db.save(foo);
-        assertEquals(201, response.getStatusCode());
+        assertEquals(2, response.getStatusCode() / 100);
     }
 
     @Test
@@ -81,7 +81,7 @@ public class ResponseTest {
     public void verifyDocumentConflict() {
         try {
             Response response = db.save(foo);
-            assertEquals(201, response.getStatusCode());
+            assertEquals(2, response.getStatusCode() / 100);
 
             db.save(foo);
             fail("A DocumentConflictException should be thrown");
@@ -111,7 +111,7 @@ public class ResponseTest {
 
         List<Response> responses = db.bulk(foos);
         for (Response response : responses) {
-            assertEquals(201, response.getStatusCode());
+            assertEquals(2, response.getStatusCode() / 100);
         }
     }
 

--- a/src/test/java/com/cloudant/tests/UnicodeTest.java
+++ b/src/test/java/com/cloudant/tests/UnicodeTest.java
@@ -267,7 +267,7 @@ public class UnicodeTest {
             conn.requestProperties.put("Accept", "application/json");
             conn.setRequestBody("{\"foo\":\"" + TESTSTRING + "\"}");
             clientResource.get().executeRequest(conn);
-            assertEquals(201, conn.getConnection().getResponseCode());
+            assertEquals(2, conn.getConnection().getResponseCode() / 100);
             closeResponse(conn);
         }
         {
@@ -302,7 +302,7 @@ public class UnicodeTest {
             conn.requestProperties.put("Accept", "application/json");
             conn.setRequestBody("{\"foo\":\"" + TESTSTRING_ESCAPED + "\"}");
             clientResource.get().executeRequest(conn);
-            assertEquals(201, conn.getConnection().getResponseCode());
+            assertEquals(2, conn.getConnection().getResponseCode() / 100);
             closeResponse(conn);
         }
         {

